TAO/TAO_IDL/include/ast_field.h

```diff
@@ -113,7 +113,12 @@ class TAO_IDL_FE_Export AST_Field : public virtual AST_Decl
 
   // Data Accessors.
   AST_Type *field_type (void) const;
-  Visibility visibility (void) const;
+
+  /// Get and Set Visibility
+  ///{
+  Visibility visibility () const;
```

Comment:
Where is this method defined?

Reply:
[Here](https://github.com/DOCGroup/ACE_TAO/blob/3507f3c21ac55d6ffe1806a1b46fcbc7113bba55/TAO/TAO_IDL/ast/ast_field.cpp#L219), it's not new, I just moved the declaration to be with its counterpart.